src/components/Cart/CartProducts/CartProducts.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div v-for="item in cart"
+       :key="item.id"
+       class="np-cartProducts__item"
+       @mouseover="turnHoverOn(item)"
+       @mouseleave="turnHoverOff(item)">
+
+    <img :src="item.imgUrl"
+         alt="Food"
+         class="np-cartProducts__item__image"/>
+
+    <ProductsDetails v-if="!item.isHoveredOn" :item="item"/>
+    <ProductsHover v-else :item="item"/>
+  </div>
+
+  <div class="np-cartProducts__item__price">
+    <p>{{ $t('totalPrice') }}</p>
+    <p>${{ priceTotal.toFixed(2) }}</p>
```

Comment:
usualy theres more then one currency in application, you should create a function that would add currency symbol depending on some global constans and then pass prices through it